src/solver/batchnorm/backward_ck.cpp

```diff
@@ -124,8 +124,8 @@ struct CKArgsBNormBwd
         return invoker_ptr->IsSupportedArgument(arg_ptr.get());
     }
 
-    std::array<ck::index_t, Rank> lens;    // inOutLengths
-    std::array<ck::index_t, Rank> strides; // inOutStrides
+    std::array<ck::index_t, Rank> lens;
+    std::array<ck::index_t, Rank> x_desc_strides;
```

Comment:
[minor][naming] I have a slight preference for `in_strides` as it's used elsewhere, or, another option would be `x_strides`. I prefer `in_strides` personally but it's your call.